Tools/CISupport/ews-build/steps.py

```diff
@@ -272,10 +272,10 @@ def get_number_of_prs_with_label(self, label, retry=0):
             except Exception as e:
                 yield self._addToLog('stdio', 'Failed to retrieve number of PRs.\n')
 
-            if attempt > retry:
+            if attempt >= retry:
                 return defer.returnValue(None)
             wait_for = (attempt + 1) * 15
-            yield self._addToLog('stdio', 'Backing off for {} seconds before retrying.\n'.format(wait_for))
+            yield self._addToLog('stdio', '\nBacking off for {} seconds before retrying.\n'.format(wait_for))
```

Comment:
can take this opportunity to use f-string in all these strings being modified.

Reply:
Thanks for the review! Welcome back 🥳